requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django>=3.1.7,<3.2.0
```

Comment:
Since you want your production code to run in a determined state, it's important to fix the version you are using.

Libraries need to be more flexible and thus use version ranges.